hieradata/development.yaml

```diff
@@ -142,6 +142,7 @@ govuk::apps::publishing_api::draft_content_store: 'http://draft-content-store.de
 govuk::apps::publishing_api::enable_procfile_worker: false
 govuk::apps::publishing_api::rabbitmq::configure_test_exchange: true
 govuk::apps::publishing_api::suppress_draft_store_502_error: '1'
+govuk::apps::publishing_api::govuk_content_schemas_path: '/var/govuk/govuk-content-schemas'
```

Comment:
No, that error message is wrong, it now uses the GovukSchemas gem, which automatically looks in the dist folder